exts/cesium.omniverse/cesium/omniverse/ui/attributes/custom_attribute_widgets.py

```diff
@@ -105,3 +106,18 @@ def custom_slider(stage, attr_name, metadata, property_type, prim_paths, *args,
         )
 
     return custom_slider
+
+
+def build_common_raster_overlay_properties():
+    with CustomLayoutGroup("Rendering"):
+        CustomLayoutProperty("cesium:alpha", build_fn=build_slider(0, 1))
+    with CustomLayoutGroup("Maximum Screen Space Error"):
+        CustomLayoutProperty("cesium:maximumScreenSpaceError")
+    with CustomLayoutGroup("Maximum Texture Size"):
+        CustomLayoutProperty("cesium:maximumTextureSize")
+    with CustomLayoutGroup("Maximum Simultaneous Tile Loads"):
+        CustomLayoutProperty("cesium:maximumSimultaneousTileLoads")
+    with CustomLayoutGroup("Sub Tile Cache Bytes"):
+        CustomLayoutProperty("cesium:subTileCacheBytes")
+    with CustomLayoutGroup("Credit Display"):
+        CustomLayoutProperty("cesium:showCreditsOnScreen")
```

Comment:
Maybe this file should be renamed to `attribute_widget_util.py`?

Reply:
Sure, we can do that. 

I left the old name because "util" essentially means "stuff." I try to avoid it, even though it's often unavoidable. NVIDIA has UsdPropertiesWidgetBuilder to help get away from the default, so maybe `cesium_properties_widget_builder.py` ultimately make sense--but it's not a class. "util" is fine for now. Maybe once the UI grows, the commonality of what's there will inform a more precise name.

Reply:
"builder" sounds good